tools/_setup_generation/items/item.py

```diff
@@ -5,18 +5,18 @@
 
 
 class Item:
-    CATEGORIES = ["tutorials", "gallery", "tips"]
+    CATEGORIES = ["fundamentals", "scenario_management", "visuals", "integration", "large_data", "gallery"]
     TYPES = ["code", "video", "article"]
 
     def __init__(self, parent_path: str, file_path: str):
-        self.parent_path = parent_path  # ex: docs/knowledge_base/tips
-        self.file_path = file_path  # ex: docs/knowledge_base/tips/charts.md or docs/knowledge_base/tips/charts/index.md
+        self.parent_path = parent_path
+        self.file_path = file_path
```

Comment:
I would keep the comments to help maintaining the code in a couple of months from now.